snippets/network-observability-netobserv-cli-install-warning.adoc

```diff
@@ -0,0 +1,15 @@
+// Text snippet included in the following assemblies:
+//
+//
+//
+// Text snippet included in the following modules:
+//
+// * observability/network_observability/network-observability-operator-release-notes.adoc
+// * observability/network_observability/netobserv_cli/netobserv-cli-install.adoc
+
+:_mod-docs-content-type: SNIPPET
+
+[WARNING]
+====
+The Network Observability CLI (netobserv cli) is temporarily unavailable and is expected to resolve with link:https://issues.redhat.com/browse/OCPBUGS-36146[OCPBUGS-36146].
```

Comment:
[Above](https://github.com/openshift/openshift-docs/pull/78321/files#diff-973c7810a7a5b24bb5b7c584b36d37727bcf80bd2b6c2ff79f050e4e661741dfR11) the parenthetical is (`oc netobserv`)
```suggestion
The Network Observability CLI (`oc netobserv`) is temporarily unavailable and is expected to resolve with link:https://issues.redhat.com/browse/OCPBUGS-36146[OCPBUGS-36146].
```

Reply:
omg, this is so stupid of me. Thanks Michael 🤦‍♀️